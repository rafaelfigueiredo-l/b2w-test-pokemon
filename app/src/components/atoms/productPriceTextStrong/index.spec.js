import ProductPriceTextStrong from "./";

describe("<ProductPriceTextStrong />", () => {
  const component = <ProductPriceTextStrong>Texto</ProductPriceTextStrong>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ProductPriceTextStrong component", () => {
    expect(wrapper.find(".productPriceTextStrong")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ProductPriceTextStrong component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".productPriceTextStrong").children()).toHaveLength(1);
  });
});
