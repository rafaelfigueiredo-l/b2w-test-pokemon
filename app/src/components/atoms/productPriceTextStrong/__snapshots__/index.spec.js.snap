// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductPriceTextStrong /> should mount correctly ProductPriceTextStrong component 1`] = `
<styled.span
  className="productPriceTextStrong"
>
  <span
    className="sc-AxjAm jUcFsX productPriceTextStrong"
  >
    Texto
  </span>
</styled.span>
`;
